package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk-v2",
   "author": "UMA Team",
-  "version": "0.15.18",
+  "version": "0.15.20",
```

Comment:
Intentional to skip 0.15.19?

Reply:
No - I was testing the new package.json `yarn bump-version:patch` which commits & updates the patch version.

Reply:
Resolved: https://github.com/across-protocol/sdk-v2/pull/377/commits/dcb7e14f5890ea4f0bd5cb329a761128d64d9838